lib/components/chart.dart

```diff
@@ -6,7 +6,7 @@ import 'package:intl/intl.dart';
 class Chart extends StatelessWidget {
   final List<Transaction> recentTransaction;
 
-  Chart(this.recentTransaction);
+  const Chart(this.recentTransaction);
```

Comment:
Tente usar `const` sempre que possível quando estiver criando Widgets, essa keyword vai permitir criar instâncias que alocam o mesmo espaço na memória, evitando assim que o Widget seja reconstruído, [mais detalhes aqui.](https://medium.com/flutter-community/the-flutter-const-keyword-demystified-c8d2a2609a80)

Reply:
Entendi, obrigado!